File: delimeter.js
const concatenate = (list, delimeter) => {
    var d = '';
    var my_string = '';
   
    for (number of list) {
        my_string += (d + String(number));
        d = delimeter;
    }
    return my_string;
}
console.log(concatenate([20, 21, 22, 24, 25, 26, 27], ", "));
console.log(concatenate([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], ". "));

File: delimeter.py
def concatenate(l, delimeter):
	d = ''
	my_string = ''
	for number in l:
		my_string += (d + str(number))
		d = delimeter
	return my_string

print(concatenate(list(range(10)), ", "))
print(concatenate(list(range(11, 23)), ". ")) 

    
    

